### PR TITLE
Django 1.6 compatibility

### PR DESCRIPTION
**Pull request by mkoistinen**
Removes the `deconstructible` decorator for Django 1.6 compatibility.

### CODE REVIEW

File: djangocms_link/validators.py [hunk 1]
```diff
@@ -4,12 +4,9 @@
 
 import re
 
-from django.conf import settings
 from django.core.validators import URLValidator
-from django.utils.deconstruct import deconstructible
 
 
-@deconstructible
```
Comment (FinalAngel) [inline djangocms_link/validators.py]: thats a new feature of django or python?

Comment (FinalAngel) [inline djangocms_link/validators.py]: ah NVM!

**Comment by czpython** [comment]
@mkoistinen be good a little note to say what was it used for? and can it be removed without any side effects? (never seen that decorator)

**Comment by mkoistinen** [comment]
I reviewed the source for the django.core.validators.URLValidator in both Django 1.6.11 and current, the @deconstructible decorator was added after 1.7.0 as part of migrations support. More info: https://docs.djangoproject.com/en/1.8/topics/migrations/#adding-a-deconstruct-method

**Comment by yakky** [comment]
:+1:

**Pull request merged**